docs/snapshotting/snapshot-editor.md

```diff
@@ -10,11 +10,39 @@ Firecracker snapshot consists of 2 files:
   devices states etc.
 - `memory` file: file with guest memory.
 
-`snapshot-editor` (currently) allows to modify only `vmstate` files only
-on aarch64 platform.
-
 ## Usage
 
+### `edit-memory` command
```

Comment:
I am not sure enumerating initial features is necessary.
It would look like this:
- [#4045](https://github.com/firecracker-microvm/firecracker/pull/4045)
  Added `snapshot-editor` tool for modifications of snapshot files.
  Initial commands/subcommands are:
    - `edit-memory` command:
      - `rebase` subcommand
    - `edit-vmstate` command:
      - `remove-regs` subcommand (aarch64 only)
    - `info-vmstate` command:
      - `version` subcommand
      - `vcpu-states` subcommand (aarch64 only)

Reply:
yeah, I mostly envisioned it as a plain text like `Added snapshot-editor tool for modifications of snapshot files allowing rebasing memory snapshot files, printing and removing aarch64 registers from the vmstate`. Listing detailed subcommands in the changelog is certainly an overkill.